proto/pocket/application/tx.proto

```diff
@@ -34,7 +34,9 @@ message MsgUnstakeApplication {
 message MsgUnstakeApplicationResponse {}
 
 message MsgDelegateToGateway {
-  string address = 1;
+  option (cosmos.msg.v1.signer) = "appAddress"; // https://docs.cosmos.network/main/build/building-modules/messages-and-queries
+  string appAddress = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding using cosmos' ScalarDescriptor to ensure deterministic deterministic encoding
```

Comment:
🐍 💼 
![Screenshot 2023-10-26 at 4 01 28 PM](https://github.com/pokt-network/poktroll/assets/1892194/4d3e591d-3409-4ee5-bdc0-a5be998423f6)

Reply:
We have gone over this before. Cosmos standard is to use the camelCase and their compiler supports it. Should I change these?

Reply:
We have been preferring to conform with protobuf convention over cosmos on this point and I think we should continue to do so, if for no other reason than consistency.

Reply:
```suggestion
  string appAddress = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application using cosmos' ScalarDescriptor to ensure deterministic encoding.
```

Reply:
Wdyt about moving comments in protobufs to be above the lines they comment on, as we do in go source? (cc @Olshansk)

I think it would improve readability, especially for long lines like these.

Reply:
Reopening this thread to change these fields, and created #119 for future work.

---

@h5law  `Question`: Can you point to where this is enforced in their proto compiler?

> We have gone over this before. Cosmos standard is to use the camelCase and their compiler supports it. Should I change these?

I followed this thread in Cosmos (https://github.com/cosmos/cosmos-sdk/issues/6826) and it looks like they disabled automatic CamelCasing when interacting with JSON (https://github.com/cosmos/cosmos-sdk/pull/6833). I realize this is Cosmos' standard, but I think we should be consistent within our own codebase and protobuf general standards.

---

@bryanchriswhite Not going to block the comment structure here but added it as a listing check in #119.